pubsub/gcloud/aio/pubsub/subscriber_client.py

```diff
@@ -61,8 +61,7 @@ def subscribe(self,
             sub_keepalive: asyncio.Future = self._subscriber.subscribe(
                 subscription,
                 self._wrap_callback(callback),
-                flow_control=flow_control,
-                scheduler=scheduler
+                flow_control=flow_control
```

Comment:
Why the removed arg?

Reply:
If we're migrating to a fully asyncio version of this, that means that the scheduler doesn't make sense here anymore. At its core, this creates a thread which manages a queue to process messages; we'll be using the asyncio event loop for that functionality instead. The user would thus choose a different event loop long before they'd touch our constructors.